src/Chart/index.tsx

```diff
@@ -43,6 +50,7 @@ export default class ReactMinimalPieChart extends Component<Props> {
     animationEasing: 'ease-out',
     cx: 50,
     cy: 50,
+    filterSegments: () => false,
```

Comment:
I would provide no default value at all and perform the filtering only if a function is provided. How do you see it?

Reply:
I have the habit of having a noop as defaultProp so I don't have to test for it and just call it on the fly but this is probably better performance wise, agreed.